### PR TITLE
UCT/API: Update the doc for the backlog param in uct_listener_params

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

File: src/uct/api/uct.h [hunk 1]
```diff
@@ -1205,7 +1205,8 @@ struct uct_listener_params {
 
     /**
      * Backlog of incoming connection requests.
-     * If not specified, SOMAXCONN, as defined in <sys/socket.h>, will be used.
+     * If not specified, each CM component will use its maximal allowed value,
+     * based on the system's setting.
```
Comment (yosefe) [inline src/uct/api/uct.h:1209]: should we specify the backlog must be positive?

Comment (alinask) [reply src/uct/api/uct.h:1209]: /**
     * Backlog of incoming connection requests. If specified, must be a positive value.
     * If not specified, each CM component will use its maximal allowed value,
     * based on the system's setting.
     */

ok?

Comment (alinask) [reply src/uct/api/uct.h:1209]: @yosefe what do you think?

Comment (yosefe) [reply src/uct/api/uct.h:1209]: ok

**Comment by alinask** [comment]
@yosefe @shamisp @tonycurtis Can you please review?

**Review by yosefe — APPROVED** [review]

**Comment by alinask** [comment]
> uct_cm.c:242: think it's better to have affirmatively-expressed error messages, e.g. "backlog value (%d) must be a positive integer"

@tonycurtis done, thanks!

**Comment by alinask** [comment]
@shamisp @tonycurtis Can you please review?

**Review by tonycurtis — CHANGES_REQUESTED** [review]
backlog needs to be a +ve integer, not just a value

**Comment by alinask** [comment]
> backlog needs to be a +ve integer, not just a value

@tonycurtis integer is somewhat of a programming term. since this is a message that is printed out to the user, IMHO it should be a non programming related term. WDYT?

**Comment by tonycurtis** [comment]
> On Sep 3, 2020, at 9:44 AM, alinask <notifications@github.com> wrote:
> 
> 
> backlog needs to be a +ve integer, not just a value
> 
> @tonycurtis <https://github.com/tonycurtis> integer is somewhat of a programming term. since this is a message that is printed out to the user, IMHO it should be a non programming related term. WDYT?
> 
“Whole number”, then?  (3.1415927 is a +ve value)
tony

**Comment by alinask** [comment]
> On Sep 3, 2020, at 9:44 AM, alinask ***@***.***> wrote: backlog needs to be a +ve integer, not just a value @tonycurtis <https://github.com/tonycurtis> integer is somewhat of a programming term. since this is a message that is printed out to the user, IMHO it should be a non programming related term. WDYT?
> “Whole number”, then? (3.1415927 is a +ve value) tony

@tonycurtis I updated. Please take a look.

**Review by tonycurtis — APPROVED** [review]

**Comment by alinask** [comment]
@shamisp Can you please review as well?

**Comment by shamisp** [comment]
👍

**Comment by shamisp** [comment]
@yosefe something weird is going on with regression system. I see multiple workers stuck on multiple PRs

**Comment by yosefe** [comment]
VM issue
bot:pipe:retest

**Review by yosefe — APPROVED** [review]

**Review by shamisp — APPROVED** [review]

**Comment by alinask** [comment]
bot:pipe:retest

**Pull request merged**